app/modules/ads/index.js

```diff
@@ -357,6 +358,21 @@ class Ads {
     }
   }
 
+  /**
+   * @private
+   */
+  trackConnectionDownlinkToDW() {
+    const { utils } = window.Wikia.adEngine;
+    const connection = navigator.connection
+      || navigator.mozConnection
+      || navigator.webkitConnection;
+
+    if (connection) {
+      pageTracker.trackProp('downlink', connection.downlink);
```

Comment:
What to you think about concatenating the info below?
- NetworkInformation.downlink (what is the distribution of network speed among users?)
- NetworkInformation.effectiveType (will make easier to create buckets for distribution of speed)
- NetworkInformation.rtt (how long it takes for a request to make a round-trip?, so I'd read it as network latency)
- NetworkInformation.saveData (how many users are using data saving?)
- NetworkInformation.type (what is the average speed of wi-fi vs cellular?)